lib/tutorials/en_US/plugins.md

```diff
@@ -1,205 +1,166 @@
+You may also pass a second optional parameter to `server.register()`. Documentation for this object can be found in the [API reference](/api#-await-serverregisterplugins-options).
 
-The options object is used by hapi and is *not* passed to the plugin(s) being loaded. It allows you to pre-select servers based on one or more labels, as well as apply `vhost` or `prefix` modifiers to any routes that your plugins register.
+The options object is used by hapi and is *not* passed to the plugin(s) being loaded. It allows you to apply `vhost` or `prefix` modifiers to any routes that your plugins register.
 
 For example, let's say we have a plugin that looks like this:
 
 ```javascript
 'use strict';
 
-exports.register = function (server, options, next) {
+exports.plugin = {
+    pkg: require('./package.json'),
+    register: async function (server, options) {
 
-    server.route({
-        method: 'GET',
-        path: '/test',
-        handler: function (request, reply) {
-            reply('test passed');
-        }
-    });
+        server.route({
+            method: 'GET',
+            path: '/test',
+            handler: function (request, h) {
 
-    next();
-};
-
-exports.register.attributes = {
-    pkg: require('./package.json')
+                return 'test passed';
+            }
+        });
+    }
 };
 ```
 
 Normally, when this plugin is loaded it will create a `GET` route at `/test`. This can be changed by using the `prefix` setting in the options, which will prepend a string to all routes created in the plugin:
 
 ```javascript
-server.register({ register: require('myplugin') }, {
-    routes: {
-        prefix: '/plugins'
-    }
-}, (err) => {
+const start = async function () {
 
-    if (err) {
-        throw err;
-    }
-});
+    await server.register(require('myplugin'), {
+        routes: {
+            prefix: '/plugins'
+        }
+    });
+};
 ```
 
 Now when the plugin is loaded, because of the `prefix` option the `GET` route will be created at `/plugins/test`.
 
-Similarly the `config.vhost` parameter will assign a default `vhost` configuration to any routes created by the plugins being loaded. More detail about the `vhost` configuration can be found in the [API reference](/api#route-options).
-
-The `select` parameter works exactly the same way as `server.select()` does, in that you may pass one label or an array of labels for the plugin to be associated with.
-
-```javascript
-server.register({ register: require('myplugin') }, {
-    select: ['webserver', 'admin']
-}, (err) => {
-
-    if (err) {
-        throw err;
-    }
-});
-```
-
-This allows you to attach a plugin to specific connections in a server without having to change the code of the plugin.
+Similarly the `options.routes.vhost` property will assign a default `vhost` configuration to any routes created by the plugins being loaded. More detail about the `vhost` configuration can be found in the [API reference](/api#-serverrouteroute).
```

Comment:
Is it worth also making a note about the `once` registration option?

Reply:
Not sure it's worth enumerating all the options in the tutorial. There's always the API docs to get more detailed reference.

Reply:
👍